Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
+RUN apt-get update
```

Comment:
Hah, I deliberately split them to build on the old cache rather the rerunning update each time. But I can see how that may cause problems in the long run.